sympy/algebras/quaternion.py

```diff
@@ -124,10 +130,10 @@ def __copysign(x, y):
 
         # Takes the sign from the second term and sets the sign of the first
         # without altering the magnitude.
-        
+
         if y == 0:
             return 0
-        return x if x*y > 0 else -x
+        return x if x * y > 0 else -x
```

Comment:
PEP8 tools keep putting spaces between `*` symbols, but sometimes it's clearer to read expressions such as `x*y`, especially when you have mixed _+_ and _-_ symbols. It depends on the case.